runtime/nls/j9vm/j9vm.nls

```diff
@@ -2448,3 +2448,59 @@ J9NLS_VM_JAVA_COMPILER_WARNING_XINT.explanation=The java.compiler system propert
 J9NLS_VM_JAVA_COMPILER_WARNING_XINT.system_action=Setting the java.compiler system property is ignored.
 J9NLS_VM_JAVA_COMPILER_WARNING_XINT.user_response=Use the -Xint option to disable the JIT.
 # END NON-TRANSLATABLE
+
+# Java 8 message for an IllegalAccessError when the class or interface can't access its super class or interface.
+# argument 1 is "class" or "interface"
+# arguments 2 and 3 are the class or interface name
+# argument 4 is " " to match Java 11 message format
+# arguments 5 and 6 are the classloader name of the class or interface
+# argument 7 is "superclass" or "superinterface"
+# arguments 8 and 9 are the super class or interface name
+# argument 10 is " " to match Java 11 message format
+# arguments 11 and 12 are the classloader name of the super class or interface
+J9NLS_VM_CLASS_LOADING_ERROR_INVISIBLE_SUPERCLASS_OR_INTERFACE_JAVA8=%1$s %3$.*2$s%4$s(from loader %6$.*5$s) cannot access its %7$s %9$.*8$s%10$s(from loader %12$.*11$s)
```

Comment:
Please keep separate messages for Java 8 and other versions.

Reply:
Updated